workspaces/adventure-pack/package.json

```diff
@@ -35,7 +35,7 @@
     "goodies:typescript:install": "yarn",
     "goodies:typescript:test": "jest --color goodies/typescript",
     "build-app": "ts-node src/scripts/package-goodies/main.ts && tsx src/scripts/build-html.tsx && webpack && cp css/style.css dist/style.css",
-    "package-goodies:test": "bash -c 'NODE_OPTIONS=--experimental-vm-modules jest --color --testPathIgnorePatterns=\"/goodies/\"'",
+    "package-goodies:test": "cross-env NODE_OPTIONS=--experimental-vm-modules jest --color --testPathIgnorePatterns=\"/goodies/\"",
```

Comment:
Yeah using `cross-env` was something I had to use in the past when I work on different `os` 😄